### ISSUE TITLE
[Bug] Automatically setting preferences assigns a score to conflicted papers

### ISSUE THREAD

**Issue by nc2y** [kohler/hotcrp#228]
When setting reviewing preferences in batch, I noticed a small bug: 
- click on select all papers
- click on update preferences
- type in -20 or whatever in the text field, and click the Go button

Now, all papers are set to a -20 preference, **including your conflicts**.

I then manually checked all my conflicts, assigned them a -100 from that bottom "update preferences" box, and confirmed this also worked. 

So, basically, one can set arbitrary preferences to conflicts. I don't know if that's a huge problem - if the algorithm for paper assignments throws out conflicts first, it shouldn't be - but one may as well want to fix it. 

Version: whatever Oakland 2021 is using. Unfortunately, I couldn't immediately find what they are running.

**Comment by kohler** [comment]
It's allowed for conflicts to have preference settings. For instance, ppl often set preferences before conflicts are finalized. The assignment algorithm does ignore conflicts.

**Comment by nc2y** [comment]
Great. I think I was just puzzled by the fact that HotCRP doesn't let you update these preference settings directly (there is no textbox in front of a given conflicted submission), but does when you use the "aggregate" update function. It looked like a bug, but I'm glad it's just cosmetic.
